lib/src/rbush.dart

```diff
@@ -458,8 +458,10 @@ class RBushBase<T> {
     final getter = sortByMinX ? getMinX : getMinY;
     if (sortByMinX) {
       node.children.sort((a, b) => a.minX.compareTo(b.minX));
-      node.leafChildren.sort((a, b) => getter(a).compareTo(getter(b)));
+    } else {
+      node.children.sort((a, b) => a.minY.compareTo(b.minY));
```

Comment:
Shouldn't it just use the `getter` instead of `if`/`else`?

Reply:
Tried `node.children.sort((a, b) => getter(a).compareTo(getter(b)));` - it doesn't work because `getter` is a `Function(T)`, so it can not be applied to `_RBushNode<T>`

Tried to replace `if else` by this:

    final g = sortByMinX ? (a) => a.minX : (a) => a.minY;
    node.children.sort((a, b) => g(a).compareTo(g(b)));

It works but it is less readable for me.

Reply:
Okay let's keep it as it is :)